docsets/Discord4J/docset.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "Discord4J",
-    "version": "2.9.3",
+    "version": "2.10.0",
     "archive": "Discord4J.tgz",
```

Comment:
You need to update the main archive as well, not just the specific version one.

Reply:
Fixed now, whoops